rllib/algorithms/algorithm_config.py

```diff
@@ -3705,6 +3721,34 @@ def items(self):
         """Shim method to help pretend we are a dict."""
         return self.to_dict().items()
 
+    @property
+    def model_config_dict(self):
+        """Defines the model configuration used.
+
+        This method combines the auto configuration `self._model_auto_keys`
+        defined by an algorithm with the user-defined configuration in
+        `self._model_config_dict`.This configuration dictionary will be used to
+        configure the `RLModule` in the new stack and the `ModelV2` in the old
+        stack.
+
+        Returns:
+            A dictionary with the model configuration.
+        """
+        return self._model_auto_keys | self._model_config_dict
+
+    @property
+    def _model_auto_keys(self) -> Dict[str, Any]:
+        """Defines the main keys for auto-model configuration.
```

Comment:
Change to: "Defines, which AlgorithmConfig properties/settings should be auto-included into `self.model_config`."